client-participation/gulpfile.js

```diff
@@ -298,15 +299,15 @@ gulp.task("embedJs", function () {
 gulp.task("index", [], function () {
   var s = gulp.src("index.html");
   var basepath = prepPathForTemplate(destRootRest);
-  var domainWhitelist = '["' + polisConfig.domainWhitelist.join('","') + '"]';
+  var domainWhitelist = '["' + polis_config.get('domainWhitelist').join('","') + '"]';
   if (devMode) {
     s = s.pipe(
       template({
         basepath: basepath,
         basepath_visbundle: basepath_visbundle_dev,
         d3Filename: "d3.js",
-        fbAppId: polisConfig.FB_APP_ID,
-        useIntercom: !isTrue(polisConfig.DISABLE_INTERCOM),
+        fbAppId: polis_config.get('fb_app_id'),
+        useIntercom: !isTrue(webpack_config.get('disable_intercom')),
```

Comment:
I believe this is supposed to be `polis_config`, not `webpack_config`.

Reply:
Good catch!

@tamas-soos-toptal,

Does metasaurus’s comment make sense?

And, are there any other “webpack_config.get” matches in our branch?

Thanks!

Chris

Reply:
@crkrenn, yes, it was a mistake and there is no other “webpack_config.get” so I fixed this one. Thanks for the catch!

Reply:
Great; Thanks @tamas-soos-toptal.